packages/editor-ui/src/components/ParameterInput.vue

```diff
@@ -421,14 +423,25 @@ const editorLanguage = computed<CodeNodeEditorLanguage>(() => {
 	return getArgument<CodeNodeEditorLanguage>('editorLanguage') ?? 'javaScript';
 });
 
-const parameterOptions = computed<INodePropertyOptions[] | undefined>(() => {
-	if (!hasRemoteMethod.value) {
-		// Options are already given
-		return props.parameter.options as INodePropertyOptions[];
-	}
+const parameterOptions = computed(() => {
+	const options = hasRemoteMethod.value ? remoteParameterOptions.value : props.parameter.options;
+	const [safeOptions, invalidOptions] = partition(
+		options ?? [],
+		(option): option is INodePropertyOptions =>
+			isINodePropertyOptions(option) && isPresent(option.value) && isPresent(option.name),
+	);
 
-	// Options get loaded from server
-	return remoteParameterOptions.value;
+	if (invalidOptions.length > 0) {
+		captureMessage('Invalid parameter options', {
```

Comment:
Do we have event deduplication on FE side (i.e. we don't report the same issue twice)? Since we do this in a computed, I have a worry this might spam even more than it does now :/

Reply:
That's why I initially only did it when fetching remote options, but then we have the downside of excluding non-remote options. Maybe I can add a `watch` that only reports once, will try soon